_posts/2016-05-25-a-simple-TDD-environment-in-haskell.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+name: simple-tdd-environment-haskell
+title: A simple TDD environment in Haskell
+date: 2016-05-25 00:01:00 +00:00
+author: Liam Griffin
+image:
+    src: /assets/img/custom/blog/2016-05-25-haskell.svg
+tags:
+- tdd
+- haskell
+---
+
+I recently tried the bowling kata in Haskell ([finished implementation here](https://github.com/Gryff/bowling-kata)), which caused me to find out how set up my environment to comfortably do TDD. Hopefully others might find this helpful to begin their journey with the language. Here’s what I used:
+
+- Haskell installation: [Haskell Platform](https://www.haskell.org/platform/). This also gives you GHCi which you can use as a REPL and type inspector.
+- IDE: Any editor would suffice, but I used [Visual Studio Code](https://code.visualstudio.com/) as they have an extension for Haskell that gave me some basic intellisense features.
```

Comment:
There's not really an IntelliSense webpage for Visual Studio Code, so I could link to the main VS [IntelliSense page](https://msdn.microsoft.com/en-us/library/hcw1s69b.aspx)